examples/hybrid/define_post_processing.jl

```diff
@@ -981,4 +991,17 @@ function custom_postprocessing(sol, output_dir)
         )
     end
     Plots.mp4(anim, joinpath(output_dir, "T.mp4"), fps = 10)
+
+    anim = @animate for Y in sol.u
+        w_phy = Geometry.WVector.(Y.c.w).components.data.:1
+        plot(
+            vec(w_phy),
+            vec(Fields.coordinate_field(Y.f).z ./ 1000);
+            xlabel = "w [m/s]",
+            ylabel = "z [km]",
+            xlims = (-0.02, 0.02),
+            legend = false,
+        )
+    end
+    Plots.mp4(anim, joinpath(output_dir, "w.mp4"), fps = 10)
```

Comment:
Could we load the solution from the exported HDF5 files? I think post-processing with `sol` is destined to be deprecated

Reply:
I didn't find a current post-processing example that uses the hdf5 files. Could you point me to an example? Maybe we can also work on it together if we want to change the post-processing now.

Reply:
If we do that I think it would be better to separate the change into a separate PR?